src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.module.css

```diff
@@ -2,27 +2,26 @@
   font-size: 13px;
 }
 
-.defaultContent {
-  margin: 15px;
-}
-
 .row {
   display: flex;
   align-items: baseline;
 }
 
 .newRowGroup {
   margin-top: 16px;
+  margin-bottom: 16px;
 }
 
 .label {
-  line-height: 1.2;
   font-weight: var(--font-weight-light);
-  text-align: right;
   font-size: 12px;
   margin-right: 10px;
 }
 
+.labelValueWrapper {
+  display: flex;
+  margin-top: 10px;
+}
 .externalLink {
   padding-top: 8px;
 }
```

Comment:
Could you check vertical spacing here:

![image](https://github.com/Ensembl/ensembl-client/assets/6834224/a37185d7-4f4a-46a8-8b9a-7947821d2851)

It does not look very much like in the XD; and I do not know how intentional that is.

Reply:
This was copied from `GeneExternalReferences.module.css`, so probably a remnant of it.